lib/features/menu/coffeeMenu/screen/coffee_menu_screen.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import '../widget/coffee_card_widget.dart';
+
+class CoffeeMenuScreen extends StatefulWidget {
+  const CoffeeMenuScreen({Key? key}) : super(key: key);
+
+  @override
+  State<CoffeeMenuScreen> createState() => _CoffeeMenuScreenState();
+}
+
+class _CoffeeMenuScreenState extends State<CoffeeMenuScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          "Coffee",
```

Comment:
Also use single-quote brackets for strings
**NOT:** `"Coffee"`
**DO:** `'Coffee'`